python/nav/web/templates/navlets/getting_started_view.html

```diff
@@ -9,10 +9,9 @@
   <div class="row">
     {% if request.account.is_admin %}
       <div class="column medium-6">
-        <a id="joyrideme" href="javascript:void(0);"
```

Comment:
Never was a fan of "javascript:void(0);", w00t!

Reply:
Yeah... why not just use the <button/> element instead 🤯